database/migrations/2021_07_19_123858_create_mismatches_table.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\Models\ImportMeta;
+
+class CreateMismatchesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('mismatches', function (Blueprint $table) {
+            $table->id();
+            $table->string('statement_guid');
+            $table->string('property_id');
+            $table->binary('wikidata_value'); // Saving as BLOB to enable storing JSON seralizations
+            $table->text('external_value');
```

Comment:
Is the text() intended here, insted of string()? The metadata doc says that the field should be length-limited.

Reply:
it is, but I anticipate these fields might be longer than we expect, so I will limit the length at the business logic level rather than the database level.